core/controllers/email_manager.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Config properties and functions for managing email notifications."""
+
+import feconf
+from platform import gae_email_services
+
+
+def send_thank_you_email(full_name, user_email):
```

Comment:
Actually, I think, better to just pass the email along to admin@oppia.org, and make it so that they can just reply to the email address. No need to send a receipt email to the sender; we can show something in the UI instead.

Reply:
Yep I agree. Function replaced by `send_email_to_admin()`, which sends the form contents as email to admin email.